pkg/namesgenerator/names-generator.go

```diff
@@ -149,6 +165,12 @@ var (
 		// Leonardo Da Vinci invented too many things to list here. https://en.wikipedia.org/wiki/Leonardo_da_Vinci.
 		"davinci",
 
+		// Donna Dubinsky, CEO and co-founder of Palm, Inc. https://en.wikipedia.org/wiki/Donna_Dubinsky
+		"dubinsky",
```

Comment:
Read through the list again; possibly I'm too strict, just want to guard against companies wanting to add founders to the list, just because they are founded a company.